<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GainOffsetClass</class>
 <widget class="QWidget" name="GainOffsetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>111</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>111</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GainOffset</string>
  </property>
  <widget class="QLabel" name="gainLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gain</string>
   </property>
  </widget>
  <widget class="QLabel" name="offsetLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Offset</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="gainEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="offsetEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="gainSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="offsetSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
